Shared/Coordinators/VideoPlayerCoordinator.swift

```diff
@@ -45,19 +45,17 @@ final class VideoPlayerCoordinator: NavigationCoordinatable {
             AppDelegate.enterPlaybackOrientation()
         }
 
-        #else
-
-        PreferenceUIHostingControllerView {
-            Group {
-                if Defaults[.VideoPlayer.videoPlayerType] == .swiftfin {
+        #else // os(iOS)
```

Comment:
Don't need these comments

Reply:
I personally like this style as I find it makes it easier to visually parse preprocessor expressions especially if they are nested, but reverted this change.